### ISSUE TITLE
bail on missing scroll

### ISSUE THREAD

**Issue by jackdempsey** [drnic/appscrolls#27]
Maybe this is just important from the position of an appscrolls developer, but as I've been testing/writing some stuff, I'm surprised that it keeps working if a scroll is missing. I would expect a fatal error "unknown scroll" and an abort. 

Is this a conscious decision to keep moving on? If not, amenable to a patch?

**Comment by drnic** [comment]
I think there is a quiet "unknown scroll" warning at the start; though you don't really see it as the rails generator fills up the console. Perhaps we could prompt for "Continue? [yN]" if it finds a scroll is missing?

Dr Nic Williams - VP Developer Evangelism
Engine Yard
The Leading Platform as a Service
Mobile: +1 415 860 2185
Skype: nicwilliams
Twitter: @drnic

**Comment by jackdempsey** [comment]
Yep, you barely see the warning then it keeps going. I like option to
Continue. Will see about taking a crack at it.

On Sun, Apr 29, 2012 at 7:48 PM, Dr Nic Williams <
reply@reply.github.com
